src/Java.Interop.Tools.JavaTypeSystem/Adapters/ApiImporterOptions.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+using System.Collections.ObjectModel;
+
+namespace Java.Interop.Tools.JavaTypeSystem;
+
+public class ApiImporterOptions
+{
+	public Collection<string> SupportedRegisterAttributes { get; } = ["Android.Runtime.RegisterAttribute"];
```

Comment:
Should this be `SupportedTypeMapAttributes`?  So that we could have a `SupportedMethodMapAttributes` (should we need one)?